R/colorMaps.R

```diff
@@ -262,11 +262,8 @@ fire_map <-
     '#fffeee',
     '#ffffff')
 
-#' @title Supplemental color maps for rasterly 
-#' @description Hex codes for the "viridis" color map.
-#' Used in setting argument `color_map` or `color_key` in \code{rasterly} or \code{rasterly} layers.
-#' This may be deprecated in the future. 
-#' @format An object of class \code{character} of length 256.
+#' @inherit color_map
+#' @rdname color_map
 #' @export
 viridis_map <- c("#440154","#440255","#450357","#450558","#45065A","#46085B","#46095D","#460B5E","#460C60","#470E61","#470F62","#471164"
```

Comment:
May want to rename this if we're removing references to `viridis` from other areas of the package.